### ISSUE TITLE
Text AntiAliasing breaks for Font Sizes >349

### ISSUE THREAD

**Issue by jhnnslmk** [mrvux/dx11-vvvv#333]
Hi vux,

it seems like Text AntiAliasing breaks for font sizes >349.
Please see attached patch. Quick fix is using Text Geometry if one doesn't need much control.

[DX11-TextAliasing.zip](https://github.com/mrvux/dx11-vvvv/files/1653006/DX11-TextAliasing.zip)

Thank you!

**Comment by mrvux** [comment]
Yes, that sounds like you reach the limit of default glyph sheet size.

There is this example in order to render large text , where you can provide your own settings for the text engine:

https://github.com/mrvux/dx11-vvvv-girlpower/tree/master/girlpower/misc/large_text
